src/main/java/forestry/farming/logic/FarmLogicCrops.java

```diff
@@ -96,20 +85,17 @@ private boolean trySetCrop(World world, IFarmHousing farmHousing, BlockPos posit
 	}
 
 	@Override
-	public Collection<ICrop> harvest(World world, BlockPos pos, FarmDirection direction, int extent) {
-		Stack<ICrop> crops = new Stack<>();
-		for (int i = 0; i < extent; i++) {
-			BlockPos position = translateWithOffset(pos.up(), direction, i);
-			IBlockState blockState = world.getBlockState(position);
-			for (IFarmable seed : seeds) {
-				ICrop crop = seed.getCropAt(world, position, blockState);
-				if (crop != null) {
-					crops.push(crop);
-					break;
-				}
-			}
+	public ItemStack getIconItemStack() {
+		return new ItemStack(Items.WHEAT);
+	}
+
+	@Override
+	public String getName() {
+		if (isManual) {
+			return "Manual Crop Farm";
```

Comment:
Do these need localization?

Reply:
If they are actually showing in any GUI, please do so.